kv/manual/backup.mdx

```diff
@@ -0,0 +1,76 @@
+import Admonition from "./_admonition.mdx";
+import backupAddBucketToDash from "./images/backup-add-bucket-to-dash.png";
+import backupGcsBucketCreate from "./images/backup-gcs-bucket-create.png";
+
+# Backups
+
+<Admonition />
+
+KV databases hosted on Deno Deploy can be continuously backed up to your own S3-compatible storage buckets. This is in
+addition to the replication and backups that we internally perform for all data stored in hosted Deno KV databases to ensure
+high availability and data durability.
+
+This backup happens continuously with very little lag, enabling _[point-in-time-recovery](https://en.wikipedia.org/wiki/Point-in-time_recovery)_
+and live replication. Enabling backup for KV databases unlocks various interesting use-cases:
+
+- Retrieving a consistent snapshot of your data at any point in time in the past
+- Running a read-only data replica independent of Deno Deploy
+- Pushing data into your favorite data pipeline by piping mutations into streaming platforms and analytical databases like Kafka, BigQuery and ClickHouse
+
+## Configuring backup to Amazon S3
+
+First, create a bucket on the AWS console with default configuration:
+
+![create backup bucket](./images/backup-bucket-create.png)
```

Comment:
It would be great to have tabs here to show both UI and CLI flows. I think this needs to be more of a step by step guide (with links to the console) for folks who have never done this before. M